### DISCUSSION TITLE
status bar dettached?

### DISCUSSION THREAD

**Discussion by gas** [akinomyoga/ble.sh#230] [Q&A] [answered]
Hi, sometimes I'm getting the full status bar somehow printed as a row of text between prompts. 
I'd like to know how to force this behaviour myself.  
Should I call a variable as $bleopt_status_bar or anything like that for echo? Thank you
![statusbar](https://user-images.githubusercontent.com/93226/189958543-4251fad5-c0e7-48ba-aec8-ff0b4b890eff.png)

update: not a bug or anything, but this is happening every time my pc wakes up from sleep and terminal app redraws.

**Comment by gas** [comment]
ok I found it 
```
printf "${bleopt_prompt_status_line@P}"
```

**Comment by akinomyoga** [comment, marked as answer]
> Hi, sometimes I'm getting the full status bar somehow printed as a row of text between prompts.
> [...]
> update: not a bug or anything, but this is happening every time my pc wakes up from sleep and terminal app redraws.

Maybe this is related to the issue reported at #142 and #154. As also described in [README of powerlevel10k](https://github.com/romkatv/powerlevel10k#horrific-mess-when-resizing-terminal-window), this is a general problem of terminal and terminal applications, which is hard to perfectly solve. Nevertheless, you might change the value of [`bleopt canvas_winch_action`](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A72-Graphics#user-content-bleopt-canvas_winch_action).

> I'd like to know how to force this behaviour myself. Should I call a variable as $bleopt_status_bar or anything like that for echo? Thank you

Currently, ble.sh doesn't provide the feature, but yes, `echo "${bleopt_prompt_status_line@P}"` would roughly work except for the extensions by ble.sh such as `\q{...}` and the text justification.
